src/docs/Ways of Working/git.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 4
+slug: /git-flow
+---
+
+# Git flow
+
+This document describes the Team's git wokflow. It is bases on the [GitHub workflow](https://docs.github.com/en/get-started/quickstart/github-flow).
+
+## Prerequisites
+
+### Note.js
+
+Install [Note.js](https://nodejs.org/en/download/)
```

Comment:
*node.js

Also node isn't required on all repos.

Maybe make a section like:
```
### (Recommended) Install node.js and git
...
```

And point to where people can get git too